File: diary-server/src/main/java/com/diary/server/DiaryServerApplication.java
package com.diary.server;

import com.diary.server.config.JwtProperties;
import com.diary.server.config.WeChatProperties;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.EnableConfigurationProperties;

@SpringBootApplication
@EnableConfigurationProperties({JwtProperties.class, WeChatProperties.class})
public class DiaryServerApplication {

    public static void main(String[] args) {
        SpringApplication.run(DiaryServerApplication.class, args);
    }
}

File: diary-server/src/main/kotlin/com/diary/server/DiaryServerApplication.kt
package com.diary.server

import com.diary.server.config.JwtProperties
import com.diary.server.config.WeChatProperties
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.EnableConfigurationProperties
import org.springframework.boot.runApplication

@SpringBootApplication
@EnableConfigurationProperties(JwtProperties::class, WeChatProperties::class)
class DiaryServerApplication

fun main(args: Array<String>) {
  runApplication<DiaryServerApplication>(*args)
}
